src/Rule/PostPersistDomainRuleInterface.php

```diff
@@ -16,4 +18,13 @@ interface PostPersistDomainRuleInterface extends RuleInterface
      * @return array|string
      */
     public function after();
+
+    /**
+     * Execute Rule
+     *
+     * @param DomainEvent $event
+     *
+     * @return mixed
+     */
+    public function execute(DomainEvent $event);
```

Comment:
I think this changes are useless. The PostPersistDomainRuleInterface and DomainRuleInterface extends the RuleInterface.

Reply:
Good catch. I remove it.